src/xo-app/dashboard/health/index.js

```diff
@@ -226,6 +317,39 @@ const ALARM_COLUMNS = [
   const getOrphanVdiSnapshots = createGetObjectsOfType('VDI-snapshot')
     .filter([ snapshot => !snapshot.$snapshot_of ])
     .sort()
+  const getControlDomainVbds = createSort(createFilter(
+    createGetObjectsOfType('VBD'),
+    createSelector(
+      createSelector(
+        createGetObjectsOfType('VM-controller'),
+        vmControllers => map(vmControllers, 'id')
```

Comment:
Don't forget to use `createCollectionWrapper` when you return a new collection at each call.